src/Microsoft.ML/LearningPipelineDebugProxy.cs

```diff
@@ -225,7 +225,12 @@ internal class PipelineItemDebugColumn
     {
         public string Name { get; set; }
         public string Type { get; set; }
-        public string SlotNames { get; set; } = string.Empty;
+        public string SlotNames { get; set; }
```

Comment:
I'm not sure I fully understand the reasoning behind this rule.  Can we add the reasoning to the rule description?  Right now it just says

> All instance fields or properties should be initialized in a constructor, not in the field

Reply:
Sure, added this